### ISSUE TITLE
Only one image per card is available for viewing full screen

### ISSUE THREAD

**Issue by DuncanLithgow** [helloworld1/AnyMemo#387]
AnyMemo version 10.7

It is possible with Mnemosyne to add more than one image as an answer. But AnyMemo will always open the first image to full screen, regardless of which image I long-press. It should open the image I press, regardless of its' place in the order of images on the card.

Use case: I sometimes use images of geometric rules for height, width, clearance etc. as comprehensive answers to detailed questions. As it stands I can only 'expand' the first image.

**Comment by helloworld1** [comment]
Emmmm, I see the limitation. I will see what I can do to lift the limitation of one image. Thank you for reporting the issue.

**Comment by DuncanLithgow** [comment]
I'm not sure I'd call it "a limit of one image" so I'll just check we understand each other...

AnyMemo does show multiple images in the 'Answer' section. It's just that I can't open them full screen. Since they are information rich I can't always see the details I need to in the image unless it's full screen.

**Comment by helloworld1** [comment]
Yes. We are on the same page. If you long click the image, only the first image will be opened. I can reproduce the issue myself.

Btw, the latest anymemo is 10.8, any reason not updating to it?

**Comment by DuncanLithgow** [comment]
Only reason is that it's not been updated yet in the f-droid repository.
